### DISCUSSION TITLE
Help:  about using neoconf.nvim

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [LazyVim/LazyVim#3662] [Q&A]
I enabled neoconf, hoping to configure lsp parameters for the specified project. I created.neoconf.json, but it doesn't seem to be recognized by neoconf. 

Does this have anything to do with lazyvim's configuration?

**Comment by dotfrag** [comment]
Have you enabled the `neoconf` extra? The plugin used to be enabled by default, but has recently moved to extras.

**Reply by iamxiaojianzheng** [reply]
Yes, I enabled it.

**Reply by dpetka2001** [reply]
Why don't you use `.lazy.lua` for project specific configurations?

**Reply by iamxiaojianzheng** [reply]
I hope to achieve the effect of only enabling vuels in Project A and volar+vtsls in Project B. How should I configure in.lazy.lua?

vuels, volar, vtsls are installed at the same time.

**Reply by dpetka2001** [reply]
Could you create a `.lazy.lua` file in both projects and then return a `nvim-lspconfig` where you define which lsp servers should be enabled? So a file in `.lazy.lua` in project A with contents
```lua
return {
  "neovim/nvim-lspconfig",
  opts = {
    servers = {
      volar = { enabled = false },
      vtsls = { enabled = false },
    },
  },
}
```
Similar in project B a `.lazy.lua` file where you just disable `vuels` server. I'm guessing this should work?

**Reply by iamxiaojianzheng** [reply]
The example code you gave is effective, thank you.

I think the Col error problem can be solved in this way. Enable vuels for the Vue2 project, use the default Settings for the Vue3 project, and turn Vuels off.
